### ISSUE TITLE
UX Request: Update Internal/External/Media Link display in the backend

### ISSUE THREAD

**Issue by VictoriaSunNIH** [NCIOCPL/cgov-digital-platform#2231]
Currently when I add an internal, external or media link to lists (and on related resources), when I add a new link, the add link field is auto-collapsed, while all of the override fields display.

![image](https://user-images.githubusercontent.com/31222859/61545102-abce4280-aa14-11e9-8985-a7147b8551fe.png)

Can we update this so the add link field isn't collapsed, and the override fields are?  Then if a user wants to add override title, description or image, they can choose to expand those fields?  It's super hard to find  link amidst ALL of the fields, especially in a long list of items!

**Comment by andyvanavery31** [comment]
If this fix is a size 2 or smaller, it will be considered for future work. If it is a greater level of effort, it will not be considered.

**Comment by sarinapadilla** [comment]
We need to figure out two things before we can prioritize and start working:
1. Which fields we want to have expanded/collapsed
2. Figure out if this is practical from a dev standpoint (LOE)

We also would want to look into other paragraphs that use node/media entity reference fields.
